common/script/roleInfo.js

```diff
@@ -5055,25 +5071,11 @@
     localprops: [
       {
         is: "property",
-        name: "aria-label",
+        name: "aria-braillelabel",
```

Comment:
git diff being confused here - paragraph gained braillelabel (disallowed) while presentation lost it's information (cf. earlier comment on "none" / https://github.com/w3c/aria/pull/1945).